### ISSUE TITLE
itk-mhd installs files (to the wrong place)

### ISSUE THREAD

**Issue by mojca** [OpenGATE/Gate#123]
I'm experiencing this problem with Gate 8.0.

I'm using different directories for:
- Gate sources
- build directory
- installation directory
- destroot directory (for `make install DESTROOT=/temporary/prefix`)

I do quite a bit of manual shuffling of the files around (creating a shell script that defines the files from Geant4, putting materials database and macro files to the right place etc.). But there is one weird aspect of running `make install DESTROOT=/temporary/prefix`. I end up with the following files:
```
./opt/local/bin/Gate
./opt/local/include/localMetaConfiguration.h
./opt/local/include/metaArray.h
./opt/local/include/metaArrow.h
./opt/local/include/metaBlob.h
./opt/local/include/metaCommand.h
./opt/local/include/metaContour.h
./opt/local/include/metaDTITube.h
./opt/local/include/metaEllipse.h
./opt/local/include/metaEvent.h
./opt/local/include/metaFEMObject.h
./opt/local/include/metaForm.h
./opt/local/include/metaGaussian.h
./opt/local/include/metaGroup.h
./opt/local/include/metaImage.h
./opt/local/include/metaImageTypes.h
./opt/local/include/metaImageUtils.h
./opt/local/include/metaIOConfig.h
./opt/local/include/metaITKUtils.h
./opt/local/include/metaLandmark.h
./opt/local/include/metaLine.h
./opt/local/include/metaMesh.h
./opt/local/include/metaObject.h
./opt/local/include/metaOutput.h
./opt/local/include/metaScene.h
./opt/local/include/metaSurface.h
./opt/local/include/metaTransform.h
./opt/local/include/metaTube.h
./opt/local/include/metaTubeGraph.h
./opt/local/include/metaTypes.h
./opt/local/include/metaUtils.h
./opt/local/include/metaVesselTube.h
./opt/local/lib/libMetaIO.a
./some/weird/build/path/gate/work/build/itk-mhd/libitkzlib.a
./some/weird/build/path/gate/work/build/itk-mhd/Utilities/itkzlib/itk_zlib_mangle.h
./some/weird/build/path/gate/work/build/itk-mhd/Utilities/itkzlib/zconf.h
./some/weird/build/path/gate/work/build/itk-mhd/Utilities/itkzlib/zlib.h
./some/weird/build/path/gate/work/build/itk-mhd/Utilities/itkzlib/zlibDllConfig.h
```

First of all: what exactly is the use of those meta library files?

But they seem relatively harmless in any case. A bigger problem are those `itkzlib` files that shouldn't be installed (they end up under `/temporary/prefix`), in particular not at that weird location. My package manager first deletes them and then complains that they are gone. Are those files even needed?

This probably only needs a trivial fix in CMake files (with a nontrivial amount of figuring out what one-liner fixes the problem).

**Comment by djboersma** [comment]
Yes, I tried to address almost the same issue (metaIO stuff gets "installed", I hadn't noticed the itkzlib thing yet) in #75  and I even promised to do something about it. I think I got the CMake incantation that seemed successful at removing the spurious metaIO "install", but it needed more testing and then I got distracted by other things. I should dig that up, address the itkzlib issue too, and then submit a PR. :)
